deploy/crds/operator.ibm.com_mongodbs_crd.yaml

```diff
@@ -56,6 +56,9 @@ spec:
               type: integer
             storageClass:
               type: string
+          required:
```

Comment:
If they are required, you need to change them from the API
https://github.com/kgcarr/ibm-mongodb-operator/blob/random-username/pkg/apis/operator/v1alpha1/mongodb_types.go#L34-L35.
Remove `omitempty` and run `make code-dev` to use `operator-sdk` to regenerate the CRD and CSV.

Reply:
added